### PR TITLE
added swapscaner adapter

### PR DESCRIPTION
**Pull request by g1nt0ki**
##### Twitter Link:
https://twitter.com/swapscanner

##### List of audit links if any:
https://docs.swapscanner.io/v/english/whitepaper/audit

##### Website Link:
https://swapscanner.io

##### Current TVL:
4.77M

##### Chain:
Klaytn

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
N/A

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
N/A

##### Short Description (to be shown on DefiLlama):
Swapscanner is a next-generation DEX Aggregator that enables users to buy tokens at the lowest price in the Klaytn Network

##### Token address and ticker if any:
SCNR(`0x8888888888885b073f3c81258c27e83db228d5f3`)

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Dexes

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
N/A

##### forkedFrom (Does your project originate from another project):
N/A

##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL of Swapscanner consists of the SCNR tokens held in governance staking and SCNR/KLAY LP

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/swapscanner/index.js exports TVL: 
        
 
 
    klaytn                    0
    klaytn-pool2              8.81 M
    klaytn-staking            6.38 M
    pool2                     8.81 M
    staking                   6.38 M
    
    total                    0

File: projects/swapscanner/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const { default: BigNumber } = require('bignumber.js')
+const sdk = require('@defillama/sdk')
+
+const chain = 'klaytn'
+const stakingAddress = '0x7c59930d1613ca2813e5793da72b324712f6899d'
+const token = '0x8888888888885b073f3c81258c27e83db228d5f3'
+const LPToken = '0xe1783a85616ad7dbd2b326255d38c568c77ffa26'
+const wKLAP = '0xd7a4d10070a4f7bc2a015e78244ea137398c3b74'
+const transformedToken = `klaytn:${token}`
+const transformedWKLPAToken = `klaytn:${wKLAP}`
```
Comment (blockswords-luther) [inline projects/swapscanner/index.js:12]:
```suggestion
const wKLAY = '0xd7a4d10070a4f7bc2a015e78244ea137398c3b74'
const transformedToken = `klaytn:${token}`
const transformedWKLAYToken = `klaytn:${wKLAY}`
```

There's a typo. `KLAP` -> `KLAY`

File: projects/swapscanner/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  }
+}
+
+module.exports = {
+  klaytn: {
+    tvl: async () => ({}),
+    staking,
+    pool2,
+  }
+}
```
Comment (blockswords-luther) [inline projects/swapscanner/index.js:55]: Can you care to enable Prettier?

**Review by blockswords-luther — APPROVED** [review]
Thanks for the update! Maybe next time you can give me review on the same PR?

**Comment by g1nt0ki** [comment]
> Thanks for the update! Maybe next time you can give me review on the same PR?

I was too lazy to make a lot of comments, you can copy my changes and put it up as new pr? I will review and approve

**Review by waynebruce0x — APPROVED** [review]

**Comment by waynebruce0x** [comment]
Not sure if @blockswords-luther is making a separate PR for this. Code looks good to me

**Comment by g1nt0ki** [comment]
> Not sure if @blockswords-luther is making a separate PR for this. Code looks good to me

I will just wait a day or two, if he doesnt put a PR, I will merge this

**Comment by blockswords-luther** [comment]
Yeah I don't think I can make stark difference for this either; Merge this or #2322 either is fine by me 👍

**Comment by g1nt0ki** [comment]
merged `https://github.com/DefiLlama/DefiLlama-Adapters/pull/2322` instead

**Pull request closed without merging**